docker/php-fpm/Dockerfile-production

```diff
@@ -0,0 +1,76 @@
+FROM php:7.2-fpm-alpine
+
+# install git for computing diffs
+RUN apk add --update git
+
+# install Composer
+COPY docker/php-fpm/docker-install-composer /usr/local/bin/docker-install-composer
+RUN chmod +x /usr/local/bin/docker-install-composer && docker-install-composer
+
+# libpng-dev needed by "gd" extension
+# icu-dev needed by "intl" extension
+# postgresql-dev needed by "pgsql" extension
+# libzip-dev needed by "zip" extension
+# autoconf needed by "redis" extension
+# freetype-dev needed by "gd" extension
+# libjpeg-turbo-dev needed by "gd" extension
+RUN apk add --update \
+    libpng-dev \
+    icu-dev \
+    postgresql-dev \
+    libzip-dev \
+    autoconf \
+    freetype-dev \
+    libjpeg-turbo-dev
+
+# "zip" extension warns about deprecation if we do not use a system library
+RUN docker-php-ext-configure zip --with-libzip
+
+# "gd" extension needs to have specified jpeg and freetype dir for jpg/jpeg images support
+RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
+
+# install necessary PHP extensions requested by Composer
+RUN docker-php-ext-install \
+    bcmath \
+    gd \
+    intl \
+    opcache \
+    pgsql \
+    pdo_pgsql \
+    zip
+
+# redis PHP extension is not provided with the PHP source and must be installed via PECL, build-base used only for installation
+RUN apk add --update build-base && pecl install redis-4.0.2 && docker-php-ext-enable redis && apk del build-base
+
+# install npm
+RUN apk add --update nodejs-npm
+
+# install postgresql to allow execution of pg_dump for acceptance tests
+RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.7/main postgresql
+
+# install locales and switch to en_US.utf8 in order to enable UTF-8 support
+# see https://github.com/docker-library/php/issues/240#issuecomment-305038173
+RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing gnu-libiconv
+ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
+ENV LC_ALL=en_US.utf8 LANG=en_US.utf8 LANGUAGE=en_US.utf8
+
+# overwrite the original entry-point from the PHP Docker image with our own
+COPY docker/php-fpm/docker-php-entrypoint /usr/local/bin/
+
+# allow overwriting UID and GID o the user "www-data" to help solve issues with permissions in mounted volumes
+RUN chown www-data:www-data /var/www/html
```

Comment:
This is duplicated with line 65

Reply:
/var/www/html is owned by root without that

Reply:
Why do you use it then? Isnt it better to first copy all the files into directory and then chown it? Still duplicate in my mind